### PR TITLE
add node (to run dredd and other node package in projects) and curl

### PR DESCRIPTION
**Pull request by mtisserant**
Changes : 

- installe node (for using dredd package in projects)
- install curl (to install node)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,14 +11,22 @@ RUN apt-get -qq update && \
...
     && rm -rf /var/lib/apt/lists/*
 
 # replace the php memcache with a fixed one
 COPY php-memcache/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb
 RUN dpkg -i /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb \
     && echo php-memcache hold | dpkg --set-selections \
     && rm /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb
+
+# install node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - && \
```
Comment (Neofox) [inline Dockerfile:24]: Isn't node included in ubuntu repositories?

File: Dockerfile [hunk 2]
```diff
@@ -11,14 +11,22 @@ RUN apt-get -qq update && \
     php7.0-mcrypt php7.0-json php7.0-curl php7.0-zip php7.0-gd \
     pdftk a2ps htmldoc mysql-client \
     php7.0-mbstring \
-    php-xdebug ssh php-common \
+    php-xdebug ssh php-common curl \
     && rm -rf /var/lib/apt/lists/*
```
Comment (Neofox) [inline Dockerfile:15]: If you remove the apt list further down, you shouldn't remove it here too. apt-get has to retrieve the list again if you do so.

File: Dockerfile [hunk 3]
```diff
@@ -11,14 +11,22 @@ RUN apt-get -qq update && \
...
 
 # replace the php memcache with a fixed one
 COPY php-memcache/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb
 RUN dpkg -i /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb \
     && echo php-memcache hold | dpkg --set-selections \
     && rm /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb
+
+# install node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - && \
+    apt-get update && \
```
Comment (Neofox) [inline Dockerfile:25]: Can't you avoid to do another `apt-get update` 2min after the first one above?
Maybe you should add the repository before the first `apt-get update` so you can only do one update

**Review by Neofox — CHANGES_REQUESTED** [review]
Some optimisations should be done for faster build

File: Dockerfile (hunk 1 above)
Comment (mtisserant) [reply Dockerfile:24]: no :/

**Pull request closed without merging**